CMakeLists.txt

```diff
@@ -85,6 +86,32 @@ add_library(rdf4cpp
         src/rdf4cpp/rdf/IRI.cpp
         src/rdf4cpp/rdf/Literal.cpp
         src/rdf4cpp/rdf/Namespace.cpp
+        src/rdf4cpp/rdf/Node.cpp
+        src/rdf4cpp/rdf/Quad.cpp
+        src/rdf4cpp/rdf/Statement.cpp
+        src/rdf4cpp/rdf/datatypes/owl/Rational.cpp
+        src/rdf4cpp/rdf/datatypes/owl/Real.cpp
+        src/rdf4cpp/rdf/datatypes/rdf/LangString.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/Boolean.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/Decimal.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/Double.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/Float.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/String.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/non_negative/NonNegativeInteger.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/non_negative/PositiveInteger.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/non_negative/UnsignedByte.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/non_negative/UnsignedInt.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/non_negative/UnsignedLong.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/non_negative/UnsignedShort.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/non_positive/NegativeInteger.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/non_positive/NegativeInteger.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/non_positive/NonPositiveInteger.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/non_positive/NonPositiveInteger.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/signed/Byte.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/signed/Int.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/signed/Integer.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/signed/Long.cpp
+        src/rdf4cpp/rdf/datatypes/xsd/integers/signed/Short.cpp
```

Comment:
these seem to be duplicates with the separate object library rdf4cpp_datatypes

Reply:
I just forgot to remove the CMakeLists.txt in datatypes, that was from a previous attempt. Oops 😅